cmake/modules/hunter_autotools_project.cmake

```diff
@@ -159,7 +174,9 @@ function(hunter_autotools_project target_name)
   #
   # C Preprocessor flags
   set(cppflags)
-  get_directory_property(defs COMPILE_DEFINITIONS)
+  get_directory_property(defs
+      COMPILE_DEFINITIONS_${_config_type}
+  )
```

Comment:
I think we should use both `COMPILE_DEFINITIONS` and `COMPILE_DEFINITIONS_<CONFIG>`